### DISCUSSION TITLE
Add light theme in GraphQL Playground

### DISCUSSION THREAD

**Discussion by itslooklike** [keystonejs/keystone#7652] [Feature requests]
now the light theme does not work, it hurts your eyes when you work in sunlight

<img width="200" alt="image" src="https://user-images.githubusercontent.com/17547664/174952690-9b5dc300-8367-49cb-bf75-fbaa64255102.png">

**Comment by Achisingh** [comment]
This is a bug in GraphQL Playground, refer to https://github.com/graphql/graphql-playground/issues/1137.

**Reply by itslooklike** [reply]
hmm, at version 1.7.20 it`s work correct (in refer link - version 1.7.12)

<img width="200" alt="image" src="https://user-images.githubusercontent.com/17547664/175248782-85eb11a8-b8b1-49bf-91ff-3fe21a634724.png">
